src/pages/phy-count-serialSum/index.js

```diff
@@ -0,0 +1,420 @@
+        priceType: item.priceType
+      }))
+    setFilteredItems(filteredItemsList)
+    setEditMode(dataList.length > 0)
+  }
+
+  const filtered = useMemo(
+    () => ({
+      ...data,
+      list: data?.list?.filter(
+        item =>
+          (item.sku && item.sku.toString().toLowerCase().includes(formik.values.search.toLowerCase())) ||
+          (item.srlNo && item.srlNo.toLowerCase().includes(formik.values.search.toLowerCase())) ||
+          (item.weight && item.weight.toString().toLowerCase().includes(formik.values.search.toLowerCase()))
+      )
+    }),
+    [formik.values.search, data]
+  )
+
+  return (
+    <FormShell
+      form={formik}
+      isInfo={false}
+      isSaved={false}
+      isCleared={false}
+      isSavedClear={false}
+      maxAccess={maxAccess}
+      resourceId={ResourceIds.PhysicalCountSerialSummary}
+      filteredItems={filteredItems}
+      previewReport={editMode}
+    >
+      <VertLayout>
+        <Fixed>
+          <Grid container spacing={2}>
+            <Grid item xs={2}>
+              <ResourceComboBox
+                endpointId={SCRepository.StockCount.qry}
+                parameters={`_startAt=0&_pageSize=1000&_params=`}
+                name='stockCountId'
+                label={_labels.stockCount}
+                valueField='recordId'
+                displayField='reference'
+                values={formik.values}
+                required
+                readOnly={formik.values.siteId}
+                maxAccess={maxAccess}
+                onChange={(event, newValue) => {
+                  formik.setFieldValue('stockCountId', newValue?.recordId)
+                  formik.setFieldValue('reference', newValue?.reference)
+                  formik.setFieldValue('date', formatDateFromApi(newValue?.date))
+                  formik.setFieldValue('siteId', '')
+
+                  if (!newValue) {
+                    setSiteStore([])
+                    setFilteredItems([])
+                    formik.setFieldValue('date', '')
+                    formik.setFieldValue('reference', '')
+                    clearGrid()
+                  } else {
+                    fillSiteStore(newValue?.recordId)
+                  }
+                }}
+                error={formik.touched.stockCountId && Boolean(formik.errors.stockCountId)}
+              />
+            </Grid>
+            <Grid item xs={2}>
+              <CustomDatePicker
+                name='date'
+                label={_labels.date}
+                value={formik.values.date}
+                readOnly={true}
+                error={false}
+              />
+            </Grid>
+            <Grid item xs={2}>
+              <CustomTextField
+                name='reference'
+                label={_labels.reference}
+                value={formik.values.reference}
+                readOnly={true}
+                error={false}
+              />
+            </Grid>
+            <Grid container item xs={12} spacing={2}>
```

Comment:
Grid container item? choose one please

Reply:
the grid container cannot have a container as child
the grid container doesnt take xs 
the grid item doesn't take spacing

if there is any confusion please let's have a call about this matter